src/unittest/python/test_api.py

```diff
@@ -1,19 +1,32 @@
 
 import unittest
+from sys import path as sys_path
+from os import path as os_path
+sys_path.append(os_path.join("src", "main", "python"))
+    
 from os import environ
-from src.main.python.api import Api
 from requests.exceptions import ConnectionError
+from api import Api
+from main import AppContext
+
 
 
 class ApiTest(unittest.TestCase):
 
     @staticmethod
     def server_is_up():
-        domain = Api().host
-        port = Api().port
+        environ_host = "atlantic.cs.pdx.edu"
+        environ_port = "8080"
 
-        from requests import get
+        environ["_API_HOST"] = environ_host
```

Comment:
we'll probably want to revert this at some point - we want to have the ability to have an enviornment variable allowing us to run the api locally

Reply:
For the deploy purpose, we changed the credential store's position. We updated the class `CredentialManage` init function has the `cxt` to get the location of the file path. So we then need pass `cxt` in the Api() call when we need it for the initialization of `CredentialManage`.